themes/startwords/layouts/article/summary.html

```diff
@@ -1,5 +1,7 @@
 <article class="article-summary">
-    {{ .Summary }}
-    <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
+    <a href="{{ .Permalink }}">
+        <p>{{ .Summary | plainify }}</p>
```

Comment:
Why do we need `plainify` here ?

Reply:
without it, things like footnotes will be rendered as `<a>` here, and we never want to render anything other than plaintext (this was actually happening with the body as data essay)